packages/c/crun/package.yml

```diff
@@ -1,22 +1,22 @@
 name       : crun
-version    : 1.16.1
-release    : 1
+version    : '1.21'
+release    : 2
 source     :
-    - https://github.com/containers/crun/releases/download/1.16.1/crun-1.16.1.tar.zst : 473968be42b35eaf9477a11855b1deaa3e1072b0604a5c20a470e2f108280afb
+    - https://github.com/containers/crun/releases/download/1.21/crun-1.21.tar.zst : d3a4ce8af89125851b2bbfe2375f4050c7eca837021043c5628172feacdb88a4
 homepage   : https://github.com/containers/crun
 license    : |
```

Comment:
This shouldn't have the `|` at the end. Can you remove it, since you're here?

Reply:
Oh, that explains the `invalid license` warning.